lib/common.ts

```diff
@@ -167,8 +182,15 @@ export function runProgram(
       })
     );
   return new Promise<void>((resolve, reject) => {
-    proc.on("close", () => resolve());
-    proc.on("error", (error) => reject(error));
+    proc.on("close", () => {
+      if (errorMessage) {
+        return reject(new Error(errorMessage));
```

Comment:
Here's the error:

```
➜  node ../../dist/bin.js
audit-ci version: 6.5.0
Yarn Berry audit report results:
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: Internal Error: audit-ci-yarn-workspace@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
    at Stream.<anonymous> (/Users/quinn/Dev/quinnturner/audit-ci/dist/common.js:132:15)
    at Stream.emit (node:events:525:35)
    at parser.onError (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/JSONStream/index.js:142:12)
    at proto.charError (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/jsonparse/jsonparse.js:90:8)
    at proto.write (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/jsonparse/jsonparse.js:154:23)
    at Stream.<anonymous> (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/JSONStream/index.js:23:12)
    at stream.write (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/through/index.js:26:11)
    at Stream.ondata (node:internal/streams/legacy:20:31)
    at Stream.emit (node:events:513:28)
    at Stream.write (/Users/quinn/Dev/quinnturner/audit-ci/node_modules/event-stream/index.js:210:12)
Emitted 'error' event on ReadlineTransform instance at:
    at ReadlineTransform.onerror (node:internal/streams/readable:785:14)
    at ReadlineTransform.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v19.3.0
```

I wouldn't say that this is the cleanest error of all time, but it is a lot better!